FluidNC/src/WebUI/WebServer.cpp

```diff
@@ -860,7 +860,7 @@ namespace WebUI {
                     //Upload write
                     //**************
                 } else if (upload.status == UPLOAD_FILE_WRITE) {
-                    vTaskDelay(1 / portTICK_RATE_MS);
```

Comment:
this previously would evaluation to 1 or 0 depending on the hardware's clock rate.  Was that intentional?